### DISCUSSION TITLE
TP4 - duda en la asignacion de punteros

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#123] [Preguntas y respuestas] [answered]
Hola! podemos asignar los punteros dentro de una funcion de esta manera?:

```C
void funcion(int *puntero1, int *puntero2)
{
    //codigo
}

int main()
{
    int a=1;
    int b=2;
    
    funcion(a, b);
    
    return 0;
}
```

**Comment by martinvilu** [comment]
¿En dónde estarían los punteros? (aparte de los que están indicados como necesarios para `funcion`)

**Reply by rinrot** [reply]
los punteros estarian definidos dentro de la funcion

**Reply by martinvilu** [reply]
Por eso, aparte de los de la función

**Reply by martinvilu** [reply]
De tu código, la llamada no es correcta.

La función no tiene nada extraño

**Reply by martinvilu** [reply, marked as answer]
```diff
int main()
{
    int a=1;
    int b=2;
-    funcion(a, b);   
+    funcion(&a, &b);
    
    return 0;
}
```
Como para que funcione, necesitarías pasar direcciones de memoria.

**Comment by martinvilu** [comment]
¿Esta cerrado este tema?
